src/lib/viewers/media/VideoBaseViewer.js

```diff
@@ -88,6 +97,22 @@ class VideoBaseViewer extends MediaBaseViewer {
         super.loadUI();
     }
 
+    /**
+     * @inheritdoc
+     */
+    loadUIReact() {
+        super.loadUIReact();
+
+        this.controls = new ControlsRoot({
```

Comment:
Do we call `this.controls.destroy()` somewhere?

Reply:
We do for MP3, but not for MP4/video. Good catch.